models/ReportContent.php

```diff
@@ -113,46 +101,34 @@ public static function canReportPost(ContentActiveRecord $post, $userId = null)
         if (Yii::$app->user->isGuest) {
             return false;
         }
-
         $user = ($userId != null) ? User::findOne(['id' => $userId]) : Yii::$app->user->getIdentity();
-
         if ($user == null || $user->super_admin) {
-            return false;
+            return true;
         }
-
         // Can't report own content
```

Comment:
This option still is kinda broken because super_admin permission overrules this no matter what you do...